presto-main/src/main/java/com/facebook/presto/operator/aggregation/BooleanMinAggregation.java

```diff
@@ -21,7 +21,7 @@
 import static com.facebook.presto.operator.aggregation.state.TriStateBooleanState.NULL_VALUE;
 import static com.facebook.presto.operator.aggregation.state.TriStateBooleanState.TRUE_VALUE;
 
-@AggregationFunction("min")
+@AggregationFunction(value = "min", alias = "every")
```

Comment:
The min/max for Boolean are defined this way. We could delete them and add the Postgres bool_or function to replace max.

Does that sound reasonable?